README.rst

```diff
@@ -69,106 +98,86 @@ section before doing so.
-`installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
-
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala-bears --pre``.
-
-Be sure to use the latest pip, the default pip from Debian doesn't support our
-dependency version number specifiers. You will have to use a
-`virtualenv <https://github.com/coala-analyzer/coala/wiki/FAQ#installation-is-failing-help>`__
-in this case.
-
-|PyPI|
+-----
 
+=====
 Usage
------
+=====
 
-Basic analysis:
+For more information about how to do basic analysis, check out the
+`coala README <https://github.com/coala-analyzer/coala#usage>`__.
 
-::
+-----
 
-    echo "print('Hi!') " >> hw.py
-    coala --files hw.py --bears SpaceConsistencyBear
+=======
+Authors
+=======
 
-Finding out what analysis routines exist:
+coala-bears is maintained by a growing community. Please take a look at the
+meta information in `setup.py <setup.py>`__ for current maintainers.
 
-::
+-----
 
-    coala -l JavaScript  # Shows bears for JS
-    coala -A  # Shows all bears
-    coala -B -b SpaceConsistencyBear  # Shows full bear documentation
+================
+Getting Involved
+================
 
-If you want to learn more about **coala-bears**, its functionality and its usage,
-please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+If you would like to be a part of the coala community, you can check out our
+`Getting Involved <http://coala.readthedocs.io/en/latest/Getting_Involved/README.html>`__
+page or ask us at our active Gitter channel, where we have maintainers from
+all over the world. We appreciate any help!
 
-Authors
--------
+We also have a
+`newcomer guide <http://coala.readthedocs.io/en/latest/Getting_Involved/Newcomers.html>`__
+to help you get started by fixing an issue yourself! If you get stuck anywhere
+or need some help, feel free to contact us on Gitter or drop a mail at our
+`newcomer mailing list <https://groups.google.com/d/forum/coala-newcomers>`__.
 
-coala-bears is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+|gitter|
 
-Getting Involved
-----------------
+-----
 
-If you want to contribute to coala-bears, please take a look at the `Getting
-Involved Information
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__.
+=======
+Support
+=======
 
-We appreciate any help! Join us on one of:
+Feel free to contact us at our `Gitter channel <https://gitter.im/coala-analyzer/coala>`__, we'd be happy to help!
 
-- `gitter <https://gitter.im/coala-analyzer/coala/>`_
-- `#coala at freenode <webchat.freenode.net/?channels=coala>`_
-- `Telegram <https://telegram.me/joinchat/AuL-lwKZ8JLFZiI6SbtQVw>`_
+You can also drop an email at our
+`mailing list <https://github.com/coala-analyzer/coala/wiki/Mailing-Lists>`__.
```

Comment:
Maybe add `If you are interested in commercial support, please contact us on the Gitter channel as well.` to encourage companies a bit? Also in the other readme.